app.js

```diff
@@ -15,14 +15,14 @@ $(document).ready(function() {
     dataType: "json",
     data: {
 
-    }
+    },
     success: searchAnimals
   }); //end ajax object
 
   function searchAnimals(sA) {
     debugger;
 
-    searchAnimals.#.#.forEach(character => {
+    sA.forEach(character => {
```

Comment:
This `sA` is referring to the data I get back from the AJAX call on success. 

You may be right, it may be `sA.[something].[something]`, but I can't tell because your AJAX call is returning an error. Your API URL is probably incomplete or incorrect. I can't get anything, even if I just type it into the browser.

You gave this name on line 22 (it can be any word you choose, as long as you use the same one on both line 22 and 25).